src/test/java/teammates/test/cases/webapi/RestoreInstructorSoftDeletedCourseActionTest.java

```diff
@@ -18,12 +18,12 @@
 
     @Override
     protected String getActionUri() {
-        return Const.ResourceURIs.INSTRUCTOR_COURSES_RESTORE;
+        return Const.ResourceURIs.BIN_COURSE;
     }
 
     @Override
     protected String getRequestMethod() {
-        return PUT;
+        return DELETE;
```

Comment:
Take a look at the submissionParams below, I don't think you need `INSTRUCTOR_ID`?

Reply:
Yup your right! Removed that